tests/e2e/testsLibrary/WorkspaceHandlingTests.ts

```diff
@@ -8,35 +8,40 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
-import { CLASSES, Dashboard } from '..';
-import { e2eContainer } from '../inversify.config';
-import { CreateWorkspace as CreateWorkspace } from '../pageobjects/dashboard/CreateWorkspace';
+import 'reflect-metadata';
+import { inject, injectable } from 'inversify';
+import { CLASSES } from '../inversify.types';
+import { Dashboard } from '../pageobjects/dashboard/Dashboard';
+import { CreateWorkspace } from '../pageobjects/dashboard/CreateWorkspace';
 import { Logger } from '../utils/Logger';
 
-const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
-const createWorkspace: CreateWorkspace = e2eContainer.get(CLASSES.CreateWorkspace);
+@injectable()
+export class WorkspaceHandlingTests {
 
-export function createAndOpenWorkspace(stack: string) {
-    test(`Open 'New Workspace' page`, async () => {
-        Logger.trace(`WorkspaceHandlingTests.createAndOpenWorkspace wait for dashboard`);
-        await dashboard.waitPage();
-        Logger.trace(`WorkspaceHandlingTests.createAndOpenWorkspace click Create workspace button`);
-        await dashboard.clickCreateWorkspaceButton();
-        Logger.trace(`WorkspaceHandlingTests.createAndOpenWorkspace wait for getting started page`);
-        await createWorkspace.waitPage();
-        Logger.trace(`WorkspaceHandlingTests.createAndOpenWorkspace click on sample ${stack}`);
-        await createWorkspace.clickOnSample(stack);
-    });
-}
+    constructor(@inject(CLASSES.Dashboard) private readonly dashboard: Dashboard, @inject(CLASSES.CreateWorkspace) private readonly createWorkspace: CreateWorkspace) {}
 
-export async function stopWorkspace(workspaceName: string) {
-    await dashboard.stopWorkspaceByUI(workspaceName);
-}
+    public createAndOpenWorkspace(stack: string) {
+        test(`Open 'New Workspace' page`, async () => {
+            Logger.trace(`WorkspaceHandlingTests.createAndOpenWorkspace wait for dashboard`);
```

Comment:
Do we really need these trace logs? I guess dashboard methods have their own logging.

Reply:
I was using it for debugging when I had issues with workspace startup. I suppose that it can be removed